lib/app/app_base.nit

```diff
@@ -122,5 +122,16 @@ fun app: App do return once new App
 # However, this value can be used in log files and communications with servers.
 fun bound_platform: String do return "none"
 
+# Test if the application was bound to a platform, if not crash
+private fun test_bound_platform
+do
+	print_error "Apps must be bound to a platform at compilation using `-m linux` or `-m android`"
```

Comment:
Windows still use the `linux` module. I'll probably change the name, or extract common code, at some point in the future.